causallearn/utils/cit.py

```diff
@@ -15,78 +15,142 @@
 chisq = "chisq"
 gsq = "gsq"
 
-class CIT(object):
-    def __init__(self, data, method='fisherz', **kwargs):
+
+def CIT(data, method='fisherz', **kwargs):
```

Comment:
hmmm, I don't like this kargs design, because users doesn't know what to input?

I think the current way is fine (for backward compatibility), and later, maybe we can change it to 
cit = FisherZ(data, args) for all caller?

Reply:
I see your point! And I am also confused about this...

Now **kwargs is placed here to support some user-defined parameters (just at the algorithm call level, e.g., pc, s.t. users don't need to edit codes inside pc). E.g.,
 - `pc(data, 0.05, kci, est_width='median')` if the user wants to use pc+kci with another kernel width (which is not supported in the old `func kci`, where all parameters are set by default.)
 - `pc(data, 0.05, kci, True, 0, -1, cache_path='/my/path/to/cache.json')` to save&load citest cache.

Now I use kwargs because the additional arguments that different methods (e.g., FisherZ, KCI) can take are different, and we have to use `func CIT` as the entrance for all callers (just for backward compatibility).

args won't work because users still don't know what to input (and even the order)? A perfect way would be to declare cit outside the algorithm call:

```python
kci_obj = KCI(data, kernelZ='Polynomial', est_width='median', cache_path='/my/path/to/cache.json')
pc(data, 0.05, kci_obj, True, 0, -1) # only take the algorithm-related parameters
```

But this is not backward compatible (for user input). So a compromise might be to still use `func CIT` as entrance, and just put more instructions (on what parameters are allowable) at CIT's comment and CIT, pc's documents?

Reply:
ic. Cool!

Reply:
@MarkDana how about add a todo here to remind us later we want to remove the kargs argument?